Integration/int_gau.pro

```diff
@@ -35,6 +38,8 @@ FUNCTION int_gau, xval, yval, NSIGMA_FIT=nsigma_fit, NSIGMA_INT=nsigma_int, NTER
       wdth: !VALUES.D_NAN, $
       ts: !VALUES.D_NAN, $
       te: !VALUES.D_NAN, $
+      upscale: 1 ,$
```

Comment:
´upscale´ is always 1 this way?

Reply:
this gets overwritten by ´scale´ later in l207, I see it's only an initialization

Reply:
Yes, this should remain as is, but I will add an explanatory comment